### ISSUE TITLE
NaN values for Fisher, Kama and Volume Profile when applied as a strategy

### ISSUE THREAD

**Issue by andybrewer** [twopirllc/pandas-ta#123]
This method of calling indicators works as expected:
```
df_fisher = df.ta.fisher()
df_kama = df.ta.kama()
df_vp = df.ta.vp()
```

But this method produces NaN values:
```
df = pd.read_csv(...)
df.ta.strategy(ta.Strategy(name='fisher', ta=[{'kind': 'fisher'}]))
df.ta.strategy(ta.Strategy(name='kama', ta=[{'kind': 'kama'}]))
df.ta.strategy(ta.Strategy(name='vp', ta=[{'kind': 'vp'}]))
```

However, all other indicators seem to work perfectly in either the direct or strategy method.

**Comment by andybrewer** [comment]
`df.ta.fisher(append=True)` also appears to return NaN

**Comment by andybrewer** [comment]
I also noticed that while `df.ta.ao()` (and others) return expected results:
```
date
2004-11-18         NaN
2004-11-19         NaN
2004-11-22         NaN
2004-11-23         NaN
2004-11-24    0.000000
                ...   
2020-09-03    0.697647
2020-09-04   -0.273382
2020-09-08   -1.356706
2020-09-09   -2.181235
2020-09-10   -2.210941
Name: AO_5_34, Length: 3980, dtype: float64
```

But `df.ta.fisher()` (+ `kama()` and `vp()`) return data without an index:
```
0            NaN
1            NaN
2            NaN
3            NaN
4       0.342828
          ...   
3975    0.284772
3976   -0.333887
3977   -0.903368
3978   -0.542006
3979    0.005482
Name: FISHERT_5, Length: 3980, dtype: float64
```

**Comment by twopirllc** [comment]
Hey @andybrewer,

I will look into ```fisher``` and ```kama``` when I get a chance.

But Volume Profile, ```vp```, is based on the following without the _value area_ calculation and does not return a time series but a range of prices.

- [Stockcharts Source](https://stockcharts.com/school/doku.php?id=chart_school:technical_indicators:volume_by_price)
- [TradingView Source](https://www.tradingview.com/wiki/Volume_Profile)
- [Trading Technologies Source](https://www.tradingtechnologies.com/blog/2013/05/15/volume-at-price/)

You can also check out Issue #74 on how to use Volume Profile.

Thanks,
KJ

**Comment by andybrewer** [comment]
Thanks, this is helpful about `vp`. I forced a `pip` install and `kama` and `fischer` are working now. Looks like it was a known issue fixed here: https://github.com/twopirllc/pandas-ta/issues/97

**Comment by twopirllc** [comment]
@andybrewer,

> Thanks, this is helpful about vp.

No worries. I try to provide all the info I use for the indicators using the ```help``` method, like ```help(ta.vp)```. If the indicator is lacking information or seems off, please let me know. 

> I forced a pip install and kama and fischer are working now. Looks like it was a known issue fixed here: #97

Yes, the up to date version is always on Github. I push to ```pip``` for large releases in general. If you want bleeding edge, you can always pull the _development_ branch and help break and test things.

Regards,
KJ
